moveit_ros/moveit_servo/CMakeLists.txt

```diff
@@ -25,13 +29,13 @@ find_package(catkin REQUIRED COMPONENTS
   std_msgs
   trajectory_msgs
 )
-find_package(Eigen3 REQUIRED)
 
 catkin_package(
   INCLUDE_DIRS
     include
   LIBRARIES
-    ${LIBRARY_NAME}
+    pose_tracking
+    ${SERVO_LIB_NAME}
```

Comment:
Did you consider making this an independent package in moveit? Isn't it worth it?